resources/js/Pages/Home.vue

```diff
@@ -226,6 +221,12 @@
                 validationError: null,
                 faqDialog: false
             }
+        },
+        setup() {
```

Comment:
Do you think this should be higher up in the object order? Below components for example? Just for visibility

Reply:
Yes, having setup at the very beginning sounds like a good idea. i thought the linter would complain and save me from deciding on my own.